src/Symfony/Component/Message/Handler/CollectionOfMessageHandlers.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message\Handler;
+
+/**
+ * Represents a collection of message handlers.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class CollectionOfMessageHandlers
```

Comment:
`MessageHandlers` or `MessageHandlerCollection` for consistency with other names

Reply:
Let's go for `MessageHandlerCollection`.